src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/M2ArgumentCollection.java

```diff
@@ -62,54 +65,72 @@
+    @Argument(fullName = "min_variants_in_pileup", optional = true, doc = "Minimum number of variant reads in pileup to be considered active region.")
     public int minVariantsInPileup = 2;
 
     /**
-     * How many standard deviations above the expected number of variant reads due to error we require for a tumor pielup to be considered active
+     * How many standard deviations above the expected number of variant reads due to error we require for tool to consider a tumor pileup active.
+     * Argument sets the z-score. Here, base qualities inform expected error rate.
+     * Providing genomic intervals of interest with -L, setting the tumor standard deviation to zero and
+     * setting minimum variants in pileup to zero forces the tool to consider all provided regions as active.
      */
-    @Argument(fullName = "tumorStandardDeviationsThreshold", optional = true, doc = "How many standard deviations above the expected number of variant reads due to error we require for a tumor pielup to be considered active.")
+    @Argument(fullName = "tumorStandardDeviationsThreshold", optional = true, doc = "How many standard deviations above the expected number of variant reads due to error we require for a tumor pileup to be considered active.")
     public int tumorStandardDeviationsThreshold = 2;
 
     /**
-     * Minimum allele fraction of variant reads in normal for a pileup to be considered inactive
+     * Minimum fraction of variant reads in normal for a pileup to be considered inactive. Applies to normal data in a tumor with matched normal analysis.
+     * For value of 0.1, at least one tenth of pileup must be variant for tool to consider it a germline variant site and therefore not a region of interest
+     * in the somatic analysis.
      */
-    @Argument(fullName = "minNormalVariantFraction", optional = true, doc = "Minimum number of reads in pileup to be considered active region.")
+    @Argument(fullName = "minNormalVariantFraction", optional = true, doc = "Minimum fraction of variant reads in normal pileup to be considered germline.")
     public double minNormalVariantFraction = 0.1;
 
     /**
-     * Only variants with tumor LODs exceeding this threshold can pass filtering.
+     * Only variants with tumor LODs exceeding this threshold will be written to the VCF, regardless of filter status.
+     * Set to less than or equal to tumor_lod. Increase argument value to reduce false positives in the callset.
+     * Default setting of 3 is permissive and will emit some amount of false-positives that
+     * filtering should filter and that allows for downstream training outside of the current workflow.
      */
-    @Argument(fullName = "tumor_lod_to_emit", optional = true, doc = "LOD threshold for emit tumor variant")
+    @Argument(fullName = "tumor_lod_to_emit", optional = true, doc = "LOD threshold to emit tumor variant to results.")
```

Comment:
I try to avoid words inputs and outputs. How about we simplify this back to "LOD threshold to emit tumor variant to VCF."?